pipeline/metadata/satellite.py

```diff
@@ -735,18 +733,19 @@ def process_satellite_lines(
     post_processed_satellite: rows of satellite scan data
     blockpage_rows: rows of blockpage data
   """
-  # PCollection[Tuple[filename,line]] x3
-  tags, blockpages, lines = lines | beam.Partition(
+  # PCollection[Tuple[filename,line]] x4
+  answer_lines, resolver_lines, blockpage_lines, row_lines = lines | beam.Partition(
```

Comment:
Can you remove the blockpages from the merged input and create a separate pcollection from its own filename instead. It should be straightforward in the caller of `process_satellite_lines`

Reply:
The caller for `process_satellite_lines` is [here](https://github.com/censoredplanet/censoredplanet-analysis/blob/abdac6e698ebbc820d531ac93aabfdcfcf682a84/pipeline/beam_tables.py#L614) in `beam_tables.run_beam_pipeline`. That method has some mixed satellite and hyperquack processing code that needs to be untangled, but I really don't want to add any conditionals on satellite filenames in there. It's also difficult to do without making changes to `_read_scan_text` that would affect hyperquack.

Reply:
Ok for now. But Satellite and Hyperquack should also be disentangled.